rs/https_outcalls/adapter/src/rpc_server.rs

```diff
@@ -433,7 +433,7 @@ impl HttpsOutcallsService for CanisterHttp {
             self.client
                 .request(http_req)
                 .await
-                .map_err(|e| format!("Failed to directly connect: {:?}", e))
+                .map_err(|e| format!("Failed to directly connect (Please note that only IPv6 access is guaranteed for applications): {:?}", e))
```

Comment:
could we add a pointer to the docs for more context?
https://internetcomputer.org/docs/references/https-outcalls-how-it-works/#ipv6-only-support

Reply:
I have mixed feelings about this.
While I certainly think that this is useful, I am a bit hesitant to hardcode a Web2 URL into the replica binary.
We use the same URL in our codebase, but for testing / scaffolding. Here it would hardcoded into the binary.

Reply:
^ this is what we say in the docs as well